handleException: anException do: aBlock
	^ aBlock value